src/cookie.rs

```diff
@@ -92,7 +92,159 @@ pub(crate) fn extract_response_cookies<'a>(
 
 /// A persistent cookie store that provides session support.
 #[derive(Default)]
-pub(crate) struct CookieStore(pub(crate) cookie_store::CookieStore);
+pub struct CookieStore(pub cookie_store::CookieStore);
```

Comment:
The field should not be `pub`, or else we expose the dependency of the exact version of `cookie_store` we are using.

Reply:
Understood. But since methods will use objects set in the `cookie_store` crate, the problem will not be solved.

Reply:
So, should the builder method used to import cookies take a `CookieStore` from the cookie_store crate? And what about exporting? @seanmonstar

Reply:
No, I would say no types from those dependencies should be exposed. We could try having some parse method for `reqwest::CookieStore`. Or this could be broken up into multiple PRs, just figuring our the exporting step first.